usr/index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
```

Comment:
This file should come under `fellowship/usr`

Reply:
will do